entries/p/proton.me.json

```
@@ -11,6 +11,7 @@
       "u2f"
     ],
     "documentation": "https://proton.me/support/two-factor-authentication-2fa",
+    "notes": "An authenticator app must be enabled to use a hardware token.",
```

`TOTP` is probably better than `An authenticatior app` since there's many different ways to generate TOTP codes.
```suggestion
    "notes": "TOTP must be enabled to use a hardware token.",
```

---

i was just staying close to their original wording, but that makes more sense.